docs/swagger/spec.yml

```diff
@@ -5952,6 +6070,12 @@ definitions:
           email:
             type: string
 
+  UserScopes:
+    description: a Map(ObjectType, Array[ActionType])
```

Comment:
I think we generally want to avoid using code in a description. "A map from API resource types to allowed actions" is probably sufficient.